frontend/src/utils/geolocation.ts

```diff
@@ -1,7 +1,10 @@
 import { LatLng } from "@/interfaces";
+import axios from "axios";
 
 type Kilometer = number;
 
+const GOOGLE_API_KEY = "AIzaSyBQtPAxW1JqFaPHYhGcpwp_gYqEifJOrAU";
```

Comment:
is this the same key that is used in the env?

Reply:
...nobody saw anyth

Reply:
Might want to revoke the key since it's now retrievable from the Git history. :/